dataprofiler/tests/profilers/profiler_options/test_structured_options.py

```diff
@@ -276,6 +280,22 @@ def test_validate(self):
         option.set({"column_null_values": None})
         self.assertEqual([], option._validate_helper())
 
+        expected_error_type = [
+            f"{optpth}.sampling_ratio must be either None or an float"
+        ]
+
+        expected_error_value = [
+            "{}.sampling_ratio must be greater than 0.0 and less than or equal to 1.0".format(
+                optpth
+            )
+        ]
+        # Test ratio is not a float
+        option.set({"sampling_ratio": 1})
+        self.assertEqual(expected_error_type, option._validate_helper())
+        # Test ratio is out of range
+        option.set({"sampling_ratio": 2.5})
+        self.assertEqual(expected_error_value, option._validate_helper())
```

Comment:
i'd recommend for a slightly more encompassing test to also test the down-side range of the bounds of the option. So you test two things, which are great:

- one right at the top end 
- another past the top end

Should add another test, to fully validate the logic, where `sampling_ratio <= 0.0`

Reply:
@joshuart let me know if you have any questions